citation.cff

```diff
@@ -0,0 +1,101 @@
+authors: 
+  -
+    affiliation: "Ontario Institute for Cancer Research"
+    family-names: Yuen
+    given-names: Denis
+    orcid: "https://orcid.org/0000-0002-6130-1021"
+  -
+    affiliation: "UC Santa Cruz Genomics Institute"
+    family-names: Cabansay
+    given-names: Louise
+    orcid: "https://orcid.org/0000-0002-1475-5979"
+  -
+    affiliation: "Ontario Institute for Cancer Research"
+    family-names: Duncan
+    given-names: Andrew
+  -
+    affiliation: "Ontario Institute for Cancer Research"
+    family-names: Luu
+    given-names: Gary
+  -
+    affiliation: "Ontario Institute for Cancer Research"
+    family-names: Hogue
+    given-names: Gregory
+  -
+    affiliation: "UC Santa Cruz Genomics Institute"
+    family-names: Overbeck
+    given-names: Charles
+  -
+    family-names: Perez
+    given-names: Natalie
+  -
+    affiliation: "UC Santa Cruz Genomics Institute"
+    family-names: Shands
+    given-names: Walt
+  -
+    affiliation: "UC Santa Cruz Genomics Institute"
+    family-names: Steinberg
+    given-names: David
+  -
+    affiliation: "UC Santa Cruz Genomics Institute"
+    family-names: Reid
+    given-names: Chaz
+  -
+    affiliation: "UC Santa Cruz Genomics Institute"
+    family-names: Olunwa
+    given-names: Nneka
+  -
+    affiliation: "UC Santa Cruz Genomics Institute"
+    family-names: Hansen
+    given-names: Richard
+  -
+    affiliation: "UC Santa Cruz Genomics Institute"
+    family-names: Sheets
+    given-names: Elizabeth
+    orcid: "https://orcid.org/0000-0002-6335-4952"
+  -
+    affiliation: "UC Santa Cruz Genomics Institute"
+    family-names: "O'Farrell"
+    given-names: Ash
+  -
+    affiliation: "Ontario Institute for Cancer Research"
+    family-names: Cullion
+    given-names: Kim
+  -
+    affiliation: "Data Sciences Platform, Broad Institute"
+    family-names: "O'Connor"
+    given-names: Brian
+    name-particle: D
```

Comment:
I'd never heard of a name-particle before and I had to look it up. :) I assume D is his middle initial, and I don't think a middle initial is a name-particle, is it? No big deal in any case.

Reply:
Not sure, the name particle seems to either be used for "van", "van der", or middle initials in practice https://github.com/search?q=name-particle&type=code but that may be the blind leading the blind. 

The GitHub cite widget doesn't obviously look wrong after merging, so I think I'll allow it for now.